scripts/apps/templates/controllers/CreateTemplateController.ts

```diff
@@ -32,8 +35,25 @@ export function CreateTemplateController(
     this.types = templates.types;
     this.createTypes = _.filter(templates.types, (element) => element._id !== 'kill');
     this.save = save;
+    this.dateTimeFields = null;
 
     activate();
+    function itemData() {
+        const _item = JSON.parse(JSON.stringify(templates.pickItemData(item)));
+
+        self.dateTimeFields?.forEach((field) => {
+            if (_item.extra[field._id]) {
+                Object.values(extensions).forEach(({activationResult}) => {
+                    if (activationResult.contributions?.customFieldTypes) {
+                        if (activationResult.contributions?.customFieldTypes[0]?.onTemplateCreate) {
```

Comment:
It has to run `onTemplateCreate` for each extension, not only the first one. Not only run it, but get the return value and patch it. As I've mentioned in another comment, you can't mutate items from inside extensions.

Also, 2 arguments are passed to  `onTemplateCreate`, when in the interface(superdesk-api.d.ts) there are no arguments defined.